.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.git
```

Comment:
Need to add .mongo here too, I guess?

Reply:
We'll need to mount .mongo as a volume for the mongodb container, it won't persist data otherwise.

Reply:
docker-compose build wouldn't run a second time on unless I added that line, and it seemed to persist fine with the line added (my user account and session token were both in the db). Can you confirm this once?

Reply:
Did you shut everything down right?
```
docker-compose down
docker network rm $(docker network ls -q)
```

Reply:
Seems to work with `.mongo` in `.dockerignore` too.

Reply:
>     docker network rm $(docker network ls -q)
Hadn't run that. But I guess since it works with `.mongo` there too, it's cool.